src/vm/gcenv.ee.cpp

```diff
@@ -149,6 +149,40 @@ static void ScanStackRoots(Thread * pThread, promote_func* fn, ScanContext* sc)
     }
 }
 
+/*
+ * Scan all GC Frames
+ */
+
+static void ScanGCFrames(Thread * pThread, promote_func* fn, ScanContext* sc)
```

Comment:
This has 6 useful lines and 25 lines duplicated from ScanStackRoots. Put the loop into ScanStackRoots directly to avoid this duplication?

Reply:
Makes sense